predictors_reliability.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>add the KFold dependency at the beginning of the notebook so we don't get this error.</p><p> </p><p>eg: <code>from sklearn.model_selection import KFold</code>
</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/santiviquez/lie-detection/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>